package.json

```diff
@@ -4,9 +4,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@davatar/react": "^1.8.1",
```

Comment:
You'll also need to update the package-lock.json for this to build correctly

Reply:
Done